### PR TITLE
ddl: fix a bug in fktest

### PR DESCRIPTION
**Pull request by zxylvlp**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: ddl/foreign_key_test.go [hunk 1]
```diff
@@ -48,26 +48,35 @@ func (s *testForeighKeySuite) TearDownSuite(c *C) {
 
 func testCreateForeignKey(c *C, ctx context.Context, d *ddl, dbInfo *model.DBInfo, tblInfo *model.TableInfo, fkName string, keys []string, refTable string, refKeys []string, onDelete ast.ReferOptionType, onUpdate ast.ReferOptionType) *model.Job {
```
Comment (zimulala) [inline ddl/foreign_key_test.go]: Using `(s *testForeighKeySuite) testCreateForeignKey(...)` can reduce the parameters.

Comment (zxylvlp) [inline ddl/foreign_key_test.go]: The other functions like this are this type.

Comment (zimulala) [inline ddl/foreign_key_test.go]: Here too many parameters.

**Comment by zimulala** [comment]
LGTM

File: ddl/foreign_key_test.go (hunk 1 above)
Comment (zxylvlp) [inline ddl/foreign_key_test.go]: It seems a little. I'll change it.

**Pull request merged**